backend/src/server.ts

```diff
@@ -0,0 +1,34 @@
+import { insertSingleRow, getRows } from './databasefunctions';
+import express = require('express');
+import { QueryResultRow } from 'pg';
+import cors from 'cors';
+import formidableMiddleware = require('express-formidable');
+
+// Create a new express application instance
+const app: express.Application = express();
+app.use(formidableMiddleware());
+app.use('*', cors());
+
+app.post('/accesstoken/new', (req, res) => {
```

Comment:
```suggestion
app.post('/accesstoken', (req, res) => {
```
I recommend URIs to uniquely identify resources, and not saying anything about which action is made upon them. The HTTP method can dictate that